Source/Icebreaker/Cards/PairUpNotificationCard.cs

```diff
@@ -22,30 +22,30 @@ public class PairUpNotificationCard
         /// </summary>
         /// <returns>An attachment to append to a message.</returns>
         /// <param name="teamName">The team name.</param>
-        /// <param name="firstPerson">The first person in the pair up.</param>
-        /// <param name="secondPerson">The second person in the pair up.</param>
+        /// <param name="sender">The first person in the pair up.</param>
+        /// <param name="recipient">The second person in the pair up.</param>
         /// <param name="botDisplayName">The bot display name.</param>
         public static Attachment GetCard(
             string teamName,
-            ChannelAccount firstPerson,
-            ChannelAccount secondPerson,
+            ChannelAccount sender,
```

Comment:
@coderkrishna Make this TeamsChannelAccount, and let the caller convert to TeamsChannelAccount, instead of doing it over and over again here.

Reply:
done